### DISCUSSION TITLE
Ability to suppress Fast Refresh warnings

### DISCUSSION THREAD

**Discussion by danielhuang** [vercel/next.js#31915] [Help]
### Describe the feature you'd like to request

I use a [babel plugin](https://github.com/gajus/babel-plugin-transform-export-default-name) to automatically generate names for anonymous arrow functions. However, `Anonymous arrow functions cause Fast Refresh to not preserve local component state.` is constantly printed to the console. Fast Refresh works as expected using the babel plugin.

### Describe the solution you'd like

I would like a configuration option to disable this warning.

### Describe alternatives you've considered

I have considered naming my functions, but anonymous functions are cleaner in my opinion.

**Comment by YangJeffrey** [comment]
I definitely agree!

**Reply by sikhote** [reply]
@timneutkens please help with why this was closed. Is there a fix on the way? Seeing this issue for very simple functions that are named, causing a lot of console spam.
